practise1.js

```diff
@@ -0,0 +1,123 @@
+﻿var practise1 = (
+	function () {
+		"use strict";
+
+		var _curry = function (func, args) {
+			if (func.length <= 1) {
+				return func;
+			}
+
+			if (args) {
+				if (func.length - 1 == args.length) {
```

Comment:
But (func.length -1) - Number and args.length - Number. And,as I understand, it is guaranteed. So can using == in this particular case increase performance? Because we avoid Type comparison, that always should be passed.